brainkeeper/src/app/services/session-service.ts

```diff
@@ -61,7 +59,7 @@ export class SessionService {
     }
 
```

Comment:
I'm missing, maybe i'm wrong, a method with indicates if a session (i guess six rounds?) is over ? I think this will be necessary for the session end screen.

Reply:
Added with commit 46c0046